File: PRIMER AÑO/TALLER/Modulo2/TEMA4/EJ03/Persona.java

package tema4;


public class Persona {
    private String nombre;
    private int dni,edad;
    
    public Persona(String nombre,int dni,int edad){
        setNombre(nombre);
        setDni(dni);
        setEdad(edad);
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getDni() {
        return dni;
    }

    public void setDni(int dni) {
        this.dni = dni;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }
    
    public String toString(){
        String aux = "Mi nombre es "+ this.getNombre() + ",mi dni es "+ this.getDni() + " y tengo " +this.getEdad() + " anios";
        return aux;
    }
    
}

File: SEGUNDO AÑO/.NET/4/ej3/Persona.cs
using System;

namespace teoria2;

public class Persona(string nombre,int edad, int dni)
{

    
    public int getEdad()
    {
        return edad;
    }
    public void Imprimir()
    {
        Console.Write($"{nombre,-20} {edad,-20} {dni,-20}");
    }
    public bool EsMayorQue(Persona p)
    {
        if(edad > p.getEdad())
        {
            return true; 
        }else return false;
    }

}
